
/opt/RestEasy/node_modules/sqlite3/lib/trace.js:28
                    throw err;
                          ^
ReferenceError: False is not defined
    at /opt/RestEasy/routes/login.js:39:36
    at Statement.errBack (/opt/RestEasy/node_modules/sqlite3/lib/sqlite3.js:16:21)
--> in Database#run('Insert Into Users (username, pass_hash, isAdmin, isActive) Values(uname, shasum.digest(\'hex\'), 0, 1)', [Function])
    at /opt/RestEasy/routes/login.js:36:9
    at Layer.handle [as handle_request] (/opt/RestEasy/node_modules/express/lib/router/layer.js:82:5)
    at next (/opt/RestEasy/node_modules/express/lib/router/route.js:110:13)
    at Route.dispatch (/opt/RestEasy/node_modules/express/lib/router/route.js:91:3)
    at Layer.handle [as handle_request] (/opt/RestEasy/node_modules/express/lib/router/layer.js:82:5)
    at /opt/RestEasy/node_modules/express/lib/router/index.js:267:22
    at Function.proto.process_params (/opt/RestEasy/node_modules/express/lib/router/index.js:321:12)
    at next (/opt/RestEasy/node_modules/express/lib/router/index.js:261:10)
    at Function.proto.handle (/opt/RestEasy/node_modules/express/lib/router/index.js:166:3)

/opt/RestEasy/node_modules/sqlite3/lib/trace.js:28
                    throw err;
                          ^
ReferenceError: False is not defined
    at /opt/RestEasy/routes/login.js:39:36
    at Statement.errBack (/opt/RestEasy/node_modules/sqlite3/lib/sqlite3.js:16:21)
--> in Database#run('Insert Into Users (username, pass_hash, isAdmin, isActive) Values(uname, shasum.digest(\'hex\'), 0, 1)', [Function])
    at /opt/RestEasy/routes/login.js:36:9
    at Layer.handle [as handle_request] (/opt/RestEasy/node_modules/express/lib/router/layer.js:82:5)
    at next (/opt/RestEasy/node_modules/express/lib/router/route.js:110:13)
    at Route.dispatch (/opt/RestEasy/node_modules/express/lib/router/route.js:91:3)
    at Layer.handle [as handle_request] (/opt/RestEasy/node_modules/express/lib/router/layer.js:82:5)
    at /opt/RestEasy/node_modules/express/lib/router/index.js:267:22
    at Function.proto.process_params (/opt/RestEasy/node_modules/express/lib/router/index.js:321:12)
    at next (/opt/RestEasy/node_modules/express/lib/router/index.js:261:10)
    at Function.proto.handle (/opt/RestEasy/node_modules/express/lib/router/index.js:166:3)

/opt/RestEasy/node_modules/sqlite3/lib/trace.js:28
                    throw err;
                          ^
ReferenceError: False is not defined
    at /opt/RestEasy/routes/login.js:39:36
    at Statement.errBack (/opt/RestEasy/node_modules/sqlite3/lib/sqlite3.js:16:21)
--> in Database#run('Insert Into Users (username, pass_hash, isAdmin, isActive) Values(uname, shasum.digest(\'hex\'), 0, 1)', [Function])
    at /opt/RestEasy/routes/login.js:36:9
    at Layer.handle [as handle_request] (/opt/RestEasy/node_modules/express/lib/router/layer.js:82:5)
    at next (/opt/RestEasy/node_modules/express/lib/router/route.js:110:13)
    at Route.dispatch (/opt/RestEasy/node_modules/express/lib/router/route.js:91:3)
    at Layer.handle [as handle_request] (/opt/RestEasy/node_modules/express/lib/router/layer.js:82:5)
    at /opt/RestEasy/node_modules/express/lib/router/index.js:267:22
    at Function.proto.process_params (/opt/RestEasy/node_modules/express/lib/router/index.js:321:12)
    at next (/opt/RestEasy/node_modules/express/lib/router/index.js:261:10)
    at Function.proto.handle (/opt/RestEasy/node_modules/express/lib/router/index.js:166:3)
